packages/openid4vc/src/openid4vc-issuer/__tests__/openid4vc-issuer.test.ts

```diff
@@ -566,7 +569,7 @@ describe('OpenId4VcIssuer', () => {
           throw new Error('Not implemented')
         },
       })
-    ).rejects.toThrow('No offered credentials match the credential request.')
+    ).rejects.toThrow('Not implemented')
```

Comment:
This is not what we're testing here, why is the error changed?

Reply:
because the order changed when doing the mapping before the callback. now the mapping fails

Reply:
but then we're breaking the flow, it should error when there's no credentials matching the credential request, not call the mapper.